internal/value/value.go

```diff
@@ -2097,21 +2096,58 @@ func TextValue(v string) textValue {
 	return textValue(v)
 }
 
+type UUIDIssue1501FixedBytesWrapper struct {
+	val [16]byte
+}
+
+func NewUUIDIssue1501FixedBytesWrapper(val [16]byte) UUIDIssue1501FixedBytesWrapper {
+	return UUIDIssue1501FixedBytesWrapper{val: val}
+}
+
+// PublicRevertReorderForIssue1501 needs for fix uuid when it was good stored in DB,
+// but read as reordered. It may happen within migration period.
+func (w UUIDIssue1501FixedBytesWrapper) PublicRevertReorderForIssue1501() uuid.UUID {
```

Comment:
I think we need to put comment in another words, it is not clear in which cases users need this. Maybe supply it with a test to show intended usage?